library/core/src/char/methods.rs

```diff
@@ -390,8 +390,9 @@ impl char {
     /// assert_eq!('❤'.escape_unicode().to_string(), "\\u{2764}");
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
+    #[rustc_const_stable(feature = "const_char_escape_unicode", since = "1.48.0")]
```

Comment:
This will be stable in 1.50, the current nightly. 1.48 is already stable.
```suggestion
    #[rustc_const_stable(feature = "const_char_escape_unicode", since = "1.50.0")]
```
https://forge.rust-lang.org/#current-release-versions

Reply:
Honestly, I wasn't sure about adding stability attributes. Does the `since` field have to apply to future releases only? Because in that case all the attributes need to be changed.

Reply:
Yes, `since` needs to apply to the release where this is stabilized, so all the attributes need to be changed.